### DISCUSSION TITLE
More demos?

### DISCUSSION THREAD

**Discussion by guda-art** [fabricjs/fabric.js#9820] [Q&A] [answered]
**Undoubtedly, this is a great work**, but there are relatively few demos on the official website, and I cannot predict whether it can do more powerful things or whether there will be more demos to showcase its strength! Thank you!

**Comment by asturur** [comment, marked as answer]
Yes @guda-art the website is a bit abandoned.
Those days i m working on a new website that will allow to publish demos and docs in an easier way.
So far i m just copying all the old content and swap it, then i can improve the content
